pom.xml

```diff
@@ -25,7 +25,9 @@
                             org.wso2.carbon.securevault.hashicorp.repository;version="${project.version}"
                         </Export-Package>
                         <Import-Package>
-                            com.bettercloud.vault.*,
+                            com.bettercloud.vault,
+                            com.bettercloud.vault.api,
+                            com.bettercloud.vault.response,
```

Comment:
better to specify the version ranges as well

Reply:
We can't specify a version range here as the hashicorp vault driver is not imported as an osgi bundle.